pkgs/data/icons/tela-icon-theme/default.nix

```
@@ -0,0 +1,34 @@
+{ fetchFromGitHub, gtk3, hicolor-icon-theme, lib, stdenv }:
```

```suggestion
{ fetchFromGitHub, gtk3, hicolor-icon-theme, jdupes, lib, stdenv }:
```
This package is too big (about 732 MB), but many files are repeated in the theme variations. Symlinks can be used to reduce the needed space, and `jdupes` can help with this (reducing to about 185 MB).